src/main/kotlin/com/ampnet/payoutservice/exception/Exceptions.kt

```diff
@@ -0,0 +1,23 @@
+package com.ampnet.payoutservice.exception
+
+class ResourceNotFoundException(val errorCode: ErrorCode, exceptionMessage: String) : Exception(exceptionMessage) {
+    companion object {
+        private const val serialVersionUID: Long = 6281563419564137017L
+    }
+}
+
+class InternalException(val errorCode: ErrorCode, exceptionMessage: String) : Exception(exceptionMessage) {
+    companion object {
+        private const val serialVersionUID: Long = -2656013978175834059L
+    }
+}
+
+class InvalidRequestException(
+    val errorCode: ErrorCode,
+    exceptionMessage: String,
+    throwable: Throwable? = null
+) : Exception(exceptionMessage, throwable) {
+    companion object {
+        private const val serialVersionUID: Long = 959044880750168366L
```

Comment:
IntelliJ shows you a warning that you don't have `serialVersionUID` defined. You resolve the issue via it's action and it generates the value for you. The value itself is not important, it can be randomly generated. But I think that IntelliJ uses the class code to calculate some hash and that provides the value.